src/js/button.js

```diff
@@ -45,6 +45,7 @@ class Button extends ClickableComponent {
     attributes = assign({
       type: 'button', // Necessary since the default button type is "submit"
       'aria-live': 'polite' // let the screen reader user know that the text of the button may change
+      'tabindex' : 0 // add tabindex for keyboard accessibility to buttons
```

Comment:
If it's already a button element, we don't want to add a `tabindex`.
Though, I guess we do probably want to add a `tabindex` if it isn't a button.
Thoughts, @OwenEdwards?